### PR TITLE
feat: ㊙️ Input passowrd custom icon

### LINKED ISSUE #23778: How to set a new eye-visible svg for Input.Password
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
I expect to replace the `eye-visible` and `eye-invisible` svg with my new designed images for Input.Password.  But I did't found any APIs to do this.

### What does the proposed API look like?
eye-visible-class:{}

eye-invisible-class:{}

### PR DESCRIPTION
**Pull request by afc163**
### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

close #23778

### 💡 Background and solution

```jsx
    <Input.Password
      placeholder="input password"
      iconRender={visible => (visible ? <EyeTwoTone /> : <EyeInvisibleOutlined />)}
    />
```

Also resolve #23484 and #15369

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | Support Input.Password `iconRender` prop. |
| 🇨🇳 Chinese | 新增 Input.Password `iconRender` 属性用于自定义切换图标 |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️ 

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Review by xrkffgg — COMMENTED** [review]
image

点了之后，移开会 保留颜色

**Comment by xrkffgg** [comment]
点击后 变色，这是新加的吗？ 😂

**Comment by afc163** [comment]
> 点了之后，移开会 保留颜色

仔细看 demo 代码。

File: components/input/demo/password-input.md [hunk 1]
```diff
@@ -14,7 +14,17 @@ title:
...
+import { Input, Space } from 'antd';
+import { EyeInvisibleOutlined, EyeTwoTone } from '@ant-design/icons';
 
-ReactDOM.render(<Input.Password placeholder="input password" />, mountNode);
+ReactDOM.render(
+  <Space direction="vertical">
+    <Input.Password placeholder="input password" />
+    <Input.Password
+      placeholder="input password"
+      iconRender={visible => (visible ? <EyeTwoTone /> : <EyeInvisibleOutlined />)}
```
Comment (afc163) [inline components/input/demo/password-input.md:25]: EyeTwoTone 自带颜色。

Comment (xrkffgg) [reply components/input/demo/password-input.md:25]: 😅 没注意这块

**Review by xrkffgg — APPROVED** [review]

File: components/input/demo/password-input.md (hunk 1 above)
Comment (ameyarumbleon) [reply components/input/demo/password-input.md:25]: IconRender does give an error saying its not a property for <Input.Password>
However the VisibilityToggle does work. Can someone please address this.

**Pull request merged**